code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -290,21 +290,18 @@
 	occupant_typecache = typecacheof(/mob/living/carbon)
 
 /obj/machinery/bci_implanter/on_deconstruction()
-	var/obj/item/organ/internal/cyberimp/bci/bci_to_implant_resolved = bci_to_implant?.resolve()
-	bci_to_implant_resolved?.forceMove(drop_location())
-	bci_to_implant = null
+	drop_stored_bci()
 
 /obj/machinery/bci_implanter/Destroy()
-	QDEL_NULL(bci_to_implant)
+	qdel(bci_to_implant)
 	return ..()
 
 /obj/machinery/bci_implanter/examine(mob/user)
-	. = ..()
-
-	if (isnull(bci_to_implant?.resolve()))
+	if (isnull(bci_to_implant))
 		. += span_notice("There is no BCI inserted.")
```

Comment:
the . operator is not defined here. It's currently null so this will fail. You need to initialize it as a new list at the start of the proc.